File: SnakeLadderJava/snakeladdersgame/Dice.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

package snakeladdersgame;
import java.util.Random;
/**
 *
 * @author Aspire4755
 */
public class Dice {
    private int sides;

    public Dice(int sides) {
        this.sides = sides;
    }

    public Dice() {
        this.sides=6;
    }
    
    public int rollDice(){
        Random r=new Random();
        int result=r.nextInt(sides)+1;
        return result;
    }
}

File: Engine/Dice.cs
namespace Engine
{
  public class Dice {
    private int sides;

    public Dice(int sides) {
        this.sides = sides;
    }

    public Dice() {
        this.sides=6;
    }
    
    public int rollDice(){
        Random rnd = new Random();
        int hasil = rnd.Next(1,7)
        
       
          if(hasil == 6){
            for(int a =2 ;a<=0;a--){
            return Player.move(hasil)++
          }
          }
          else{
          return Player.move();
        }
          return Player.move(hasil);
        }
        
    }
}

